management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java

```diff
@@ -65,10 +65,10 @@ public Capability getCapability() {
   @Override
   public Map<String, Statistic<?, ?>> collectStatistics(Context context, Collection<String> statisticNames, long since) {
     Map<String, Statistic<?, ?>> statistics = new HashMap<String, Statistic<?, ?>>(statisticNames.size());
-    EhcacheStatistics ehcacheStatistics = (EhcacheStatistics) findExposedObject(context);
+    StandardEhcacheStatistics ehcacheStatistics = (StandardEhcacheStatistics) findExposedObject(context);
     if (ehcacheStatistics != null) {
       for (String statisticName : statisticNames) {
-        statistics.putAll(ehcacheStatistics.queryStatistic(statisticName, since));
+        statistics.put(statisticName, ehcacheStatistics.queryStatistic(statisticName, since));
```

Comment:
Let me look into re-implementing those stats.  That is my next step.  I still have to add some stats that were removed and some new stats that Anthony requested.  Unfortunately this PR only includes all the changes that were required to fix the broken unit tests and bugs.